chart/values.yaml

```diff
@@ -99,6 +105,50 @@ resources: { }
 #   cpu: 100m
 #   memory: 128Mi
 
+deployment:
+  # -- Readiness Probes
+  readinessProbe:
+    failureThreshold: 3
+    periodSeconds: 5
+    successThreshold: 1
+    timeoutSeconds: 10
+    initialDelaySeconds: 30
```

Comment:
can you remove this part?

```
    failureThreshold: 3
    periodSeconds: 5
    successThreshold: 1
    timeoutSeconds: 10
    initialDelaySeconds: 30
```
30 sec us much too high for this application as it starts fast.

The kubernetes default should be enough
https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes

If someone needs it changed, it can be overwritten in the values.

Reply:
addressed and PR is for review now, many thanks